src/routes/ViewAllLogs/ViewAllLogs.js

```diff
@@ -125,13 +122,39 @@ class ViewAllLogs extends Component {
     window.open(path, '_blank').focus();
   }
 
+  renderFilters = onChange => {
+    const { resources } = this.props;
+
+    const jobProfiles = get(resources, ['records', 'records'], [])
+      .map(item => item.jobProfileInfo);
+    const users = get(resources, ['records', 'records'], [])
+      .map(item => {
+        return {
```

Comment:
Why not to use 
.map(item => ({
  userId: item.userId,
  firstName: item.runBy.firstName,
  lastName: item.runBy.lastName,
}));